src/Microsoft.DotNet.Wpf/src/PresentationCore/System/Windows/Input/InputScopeConverter.cs

```diff
@@ -109,7 +109,12 @@ public override object ConvertFrom(ITypeDescriptorContext context, CultureInfo c
 
             if (null != stringSource)
             {
+#if NET
                 ReadOnlySpan<char> spanSource = stringSource;
```

Comment:
This implict cast is missing Eric? We should define it somewhere central to avoid #if like this.